### ISSUE TITLE
Unable to mine... Cant sign block template with CPID  - Error

### ISSUE THREAD

**Issue by sunk818** [biblepay/biblepay#80]
Using 1.1.9.2, I am not sure if solo mining is working. 
purepool has same error if wallet has no CPID.
have not tried with pool.biblepay.org

biblepay.conf:

```json
addnode=node.biblepay.org
gen=1
genproclimit=1
nickname=xxx
```

```css
13:48:28 getmininginfo
13:48:28 {
  "blocks": 103165,
  "blocks_tip": 103165,
  "currentblocksize": 1000,
  "currentblocktx": 0,
  "difficulty_podc": 1082.042374204401,
  "difficulty_pow": 1476.130394615283,
  "difficulty": 1082.042374204401,
  "errors": "",
  "genproclimit": 1,
  "networkhashps": 322945.1148321619,
  "hashps": 31.49346537040253,
  "minerstarttime": "02-22-2019 06:11:43",
  "hashcounter": 1768380,
  "pooledtx": 1,
  "testnet": false,
  "chain": "main",
  "biblepay-generate": true,
  "poolinfo1": "Unable to mine... Cant sign block template with CPID  - Error ; ",
  "poolinfo2": "Failed to sign CPID Signature (Address does not refer to key); ",
  "poolinfo3": "",
  "podc_tried": 0,
  "podc_sent": 0,
  "podc_sent_amount": 0,
  "podc_average_amount_sent": 0,
  "miningpulse": 8781,
  "poolmining": false,
  "pool_url": "",
  "poolmining_use_ssl": false,
  "pool_high_tithe": 2800,
  "pool_my_total_tithes": 850.9929512,
  "pool_total_tithes": 13577.68814117,
  "pog_difficulty": 33235.79901064118,
  "pog_min_coin_age": 30.55,
  "pog_min_coin_amount": 12679.14,
  "pog_max_tithe_amount": 5.05999999
}
```

**Comment by biblepay** [comment]
Yes, this is kind of interesting.  So this is happening in one circumstance while PODC is still enabled, and would have stopped If we transitioned to retire PODC, but obviously we need to be able to mine right now in this circumstance.

The CPID signature is technically not required in the block now anyway, so in this next leisure release Ill disable this check.

It should be out by midnight tonight.

**Comment by sunk818** [comment]
So does this mean botnet can join now?

**Comment by biblepay** [comment]
@sunk818 Not really, the botnet was attracted to us for the financial reward of : being rewarded the full POBH subsidy (not 4%) for one, and that was profitable enough to make horizontal mounted botnet.  I look at this differently now:  If they want to mine for 110 bbp please come and give us the extra hash power.

We called them a botnet because they appeared to take all of our rewards and never wanted to upgrade.

**Comment by biblepay** [comment]
Should be fixed in 1.1.9.3.
